app/ui/src/app/integration/edit-page/flow-view/flow-view-step.component.scss

```diff
@@ -103,9 +99,8 @@
       background: #ec7a08;
       border-radius: 50%;
       padding: 5px;
-      top: -6px;
-      right: -6px;
       cursor: pointer;
+      transform: translate(100%, -100%);
```

Comment:
This positions the icon from the center of the connection icon. That's one way of doing it, but it's dependent upon the size of the connection icon and warning icon. A better way would do it is to assign `top: 0; right: 0; transform: translate(25%,-25%);`. It's always a good idea to add a vertical (`top`, `bottom`) and horizontal (`left`, `right`) positioning to non-`static` positioned elements.

Reply:
Thanks for the tip @michael-coker! Correct settings applied to icons now.